src/components/modal/_modal.scss

```diff
@@ -0,0 +1,57 @@
+.modal {
+    display: none; 
+    position: fixed; 
+    z-index: 200; 
+    left: 0;
+    top: 0;
+    width: 100%; 
+    height: 100%; 
+    overflow: auto; 
+    background-color: rgb(31,31,31); 
+    background-color: rgba(31,31,31,0.75); 
+
+  
+    .modal-content {
+        margin: 15% auto; 
+        padding: 40px;
+        width: 420px;	
+        background-color: #FFFFFF; 
+        position: relative;
+
+        h2 {
+            color: #000000;	
+            font-family: Verdana;	
+            font-size: 24px;	
+            font-weight: bold;	
+            letter-spacing: -0.9px;	
+            line-height: 29px;
+            margin-top: 0px;
+        }
+
+        p {
+            color: #000000;	
+            font-family: Verdana;	
+            font-size: 16px;	
+            letter-spacing: -0.5px;	
+            line-height: 24px;
+        }
+  
+        .close {
+            position: absolute;
+            right: -16px;
+            top: -16px;
+            height: 32px;
+            width: 32px;
+            border-radius: 16px;
+            border-width: 0;
+            color: #000000;
+            float: right;
+            font-size: 18px;
+            font-weight: bold;
+            box-shadow: 0 0 4px 0 rgba(0,0,0,0.3);
```

Comment:
There's a Sass mixin for this same box shadow, please use that instead. You'll have to add the following at the top of your Sass file for variables and mixins to work:

```
@import '../../scss/mixins';
@import '../../scss/variables';
```

The box shadow mixin can be used by doing:

```
.modal{
    &__close-btn{
        @include medium-shadow;
    }
}
```

(I've POC converted the close button to BEM as per a previous comment).

*Minor additional note. I don't think you need `float:right` if you're using absolute positioning.